kore/test/Test/Kore/Unification/SubstitutionNormalization.hs

```diff
@@ -125,6 +125,10 @@ test_normalize =
             [(x, mkAnd (mkVar y) a), (y, mkAnd (mkVar x) b)]
             []
             [(x, mkAnd (mkVar y) a), (y, mkAnd (mkVar x) b)]
+        , test "two cycles"
+            [(x, f (mkVar x)), (y, g (mkVar y)), (z, c)]
+            [(z, c)]
+            [(x, f (mkVar x)), (y, g (mkVar y))]
```

Comment:
It's out of the scope of this PR, but I think the tests from this file are a little harder to read than our other tests where we make it explicit for each test which are the expected and actual values. For the tests here we also have 2 expected values and it's harder to keep track of what is what. This is just an observation, though.